nixos/doc/manual/development/meta-attributes.section.md

```diff
@@ -23,15 +23,17 @@ file.
 
   meta = {
     maintainers = with lib.maintainers; [ ericsagnes ];
-    doc = ./default.xml;
```

Comment:
Looks like these generated xmls weren't checked in CI then? Good riddance. Although.. they may have diverged :confused: